### PR TITLE
[frax-finance] Add veFXS to Frax Finance strategy

### PR DESCRIPTION
**Pull request by FortisFortuna**
The main difference here is the addition of a balanceOf(address) for the veFXS contract address at 0xc8418aF6358FFddA74e09Ca9CC3Fe03Ca6aDC5b0

Thanks

### CODE REVIEW

File: test/index.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ const example = require(`../src/strategies/${strategy}/examples.json`)[0];
   console.time('getScores');
   try {
     const scores = await snapshot.utils.getScores(
-      'yam.eth',
+      'frax-finance',
```
Comment (bonustrack) [inline test/index.ts:29]: Can you remove the changes on this file?

File: src/strategies/frax-finance/note.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+TO TEST 
+npm install
+OPTIONAL
+ADD ,
+        "snapshot": '11833575' // change block number
+TO examples.json
+npm run test
```
Comment (bonustrack) [inline src/strategies/frax-finance/note.txt:7]: Can you remove this file? You can use "npm run test --strategy=frax-finance" to test your strategy. You don't need to do changes in the files that are not part of the strategy.

File: src/strategies/frax-finance/examples.json [hunk 3]
```diff
@@ -16,8 +19,7 @@
         "network": "1",
         "addresses": [
             "0x36A87d1E3200225f881488E4AEedF25303FebcAe",
-            "0x8a97a178408d7027355a7ef144fdf13277cea776"
-        ],
-        "snapshot": 11818851
```
Comment (bonustrack) [inline src/strategies/frax-finance/examples.json:21]: There should be a snapshot defined in the example file here

Comment (FortisFortuna) [reply src/strategies/frax-finance/examples.json:21]: done

File: src/strategies/frax-finance/note.txt (hunk 2 above)
Comment (FortisFortuna) [reply src/strategies/frax-finance/note.txt]: done

File: test/index.ts (hunk 1 above)
Comment (FortisFortuna) [reply test/index.ts]: done

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**